packages/internal-test-helpers/lib/get-all-property-names.ts

```diff
@@ -1,9 +1,10 @@
-export default function getAllPropertyNames(Klass: Function) {
+export default function getAllPropertyNames<T>(Klass: { prototype: T }): Set<keyof T & string> {
```

Comment:
What is the `& string` doing here? 🤔 Is it just for the sake of excluding non-`string` keys, e.g. symbols? If so, mind leaving a comment to that effect?

Reply:
Yes, there is code downstream that expects a string.